discovery/README.md

```diff
@@ -12,6 +12,9 @@ Docker Swarm comes with multiple Discovery backends.
 
 ### Hosted Discovery with Docker Hub
 
+This service is hosted by Docker Hub and requires all nodes to have
```

Comment:
@MHBauer try this (ping @vieux for tech check) It is wordier but I think it is clearer

---> https://gist.github.com/moxiegirl/fda81731b94925ff1fef

You use a hosted discovery service with Docker Swarm. The service maintains a list of IPs in your swam.  There  are several commercial services `etcd`, `consul` and `zookeeper` depending on what is best suited for your environment. You can even use a static file.  Docker Hub also provides a hosted discovery service which you can use.  

This example uses the hosted discovery service on Docker Hub. Using Docker Hub's hosted discovery service requires that each node in the swarm is connected to the internet.  To create your swarm: